src/Resources/config/services/refund.xml

```diff
@@ -17,7 +17,7 @@
             <argument type="service" id="bitbag_sylius_mollie_plugin.calculator.refund.payment_refund_calculator" />
         </service>
         <service id="bitbag_sylius_mollie_plugin.refund.order_refund" class="BitBag\SyliusMolliePlugin\Refund\OrderRefund">
-            <argument type="service" id="sylius_refund_plugin.command_bus" />
+            <argument type="service" id="sylius.command_bus" />
```

Comment:
What's the reason of changing this?

Reply:
This service is not registered in previous command bus. It provides error: no handler for message.

Reply:
Okay :)